index.html

```diff
@@ -629,7 +629,101 @@ <h3>Presentations</h3>
 that are often, but not required to be, related.
         </p>
 
-      </section>
+    </section>
+    <section>
+      <h3>Lifecycle</h3>
+
+      <em>This section is non-normative.</em>
+
+      <p>
+This section seeks to illuminate the lifecycle of a credential. Credentials 
+allow one to make statements about a subject. Moreso, it provides a succint 
+composition mechanism for tamper-evident claims that provide cryptographic 
+guarantees on the provenance of those claims. Presentations enable subjects to 
+use one or more credentials to satisfy requests from a verifier and assert 
+proof of control of those credentials. Credentials, when in motion, form a 
+general flow. This flow provides the bedrock for the lifecycle of a 
+credential, it highlights the issuance of credentials to their verification.
+      </p>
+
+      <p>
+The lifecycle of credentials will take a common path: issuance
+of claims to a credential subject, storage of credentials into credential
+repository engineered to store verifiable credentials, and then the
+presentation to a verifer. In <a href="#ex-1-example-verifiable-credential">
```

Comment:
I would like this better if there was mention of composing credentials into a presentation, because credentials are not always simply presented.
Perhaps: "and then composition of credentials into a presentation to a verifier."